packages/elvis/CHANGELOG.json

```diff
@@ -2,6 +2,27 @@
   "$schema": "../components/changelogSchema.json",
   "name": "elvis",
   "content": [
+    {
+      "version": "19.0.0",
+      "date": "June 6, 2024",
```

Comment:
Remember to update the date before merge here too

Also, remember to include the changes of https://github.com/3lvia/designsystem/pull/2917 and https://github.com/3lvia/designsystem/pull/2911